python/paddle/v2/fluid/initializer.py

```diff
@@ -1,7 +1,13 @@
 import framework
 import numpy as np
 
-__all__ = ['Constant', 'Uniform', 'Normal', 'Xavier']
+__all__ = ['Constant', 'Uniform', 'Normal', 'Xavier', 'set_random_seed']
```

Comment:
`set_random_seed` seems a private function, if it is, do not expose it.

Reply:
fixed.